### DISCUSSION TITLE
Mandar mensajes masivos

### DISCUSSION THREAD

**Discussion by Armenta-design** [codigoencasa/builderbot#1070] [General]
Hay forma de mandar un mensaje masivo a todos los registrados en el whastapp a través del bot o requiero algún complemento extra?

Si, como mandar un recordatorio "Hola, no olvides que hoy tenemos el 50% de descuento ..."

Datos de mi versión:

y las imagenes, no me cargan, marca OBJECT OBJECT

 "name": "base-bailey-mongo",
            "version": "1.0.0",
            
            Gracias

**Comment by EmmanuelMarti20** [comment]
hola lo puedes hacer con funciones dentro de un addaction

**Comment by montserrat123455** [comment]
Hola _[![image](https://github.com/user-attachments/assets/19ecdf24-42b6-4d46-8b7b-e66ca04a2ab9)]()_

**Reply by montserrat123455** [reply]
Creo en la bondad es absurdo fingir sentimientos eso no puede hacerte feliz solo es baja frecuencia que no puede alcanzar el néctar siempre irá a su lugar "personaje mosca "
